src/wise_psf.jl

```diff
@@ -0,0 +1,39 @@
+using FITSIO
+
+function wise_psf(band; sidelen=-1)
+
+  # don't believe any sensible normalization is in place, need to fix that ...
+
+  # could potentially add checks on the input values ...
+
+  # figure out where the PSF model FITS file is located
+  psf_dir = ENV["WISE_ETC"]
+
+  psf_fname_dict = [1=>"psf_coeff-w1.v1.fits",
+                    2=>"psf_coeff-w2.v1.fits",
+                    3=>"psf_coeff-big.fits",
+                    4=>"psf_coeff-taper-w4.fits" ]
+
+  psf_fname = psf_fname_dict[band]
+  psf_fname = string(psf_dir, "/", psf_fname)
+  
+  f = FITS(psf_fname)
+  psf_model = read(f[1])
+
+  # for now just return the zeroth order term, add higher orders later
+  psf_model = psf_model[:,:,1]
```

Comment:
Yeah, each pixel value in the PSF cutout that's currently being returned is the zeroth order term of a polynomial model for that pixel's relative brightness as a function of position on the detector.